workflow/src/main/java/bio/terra/tanagra/workflow/FlattenHierarchy.java

```diff
@@ -0,0 +1,154 @@
+package bio.terra.tanagra.workflow;
+
+import avro.shaded.com.google.common.collect.Lists;
```

Comment:
This dep looks suspect, is this something for Beam compatibility, or just an accidental import?